filebeat/modules.d/tidb.yml.disabled

```diff
@@ -0,0 +1,34 @@
+# Module: tidb
+# Docs: https://www.elastic.co/guide/en/beats/filebeat/master/filebeat-module-tidb.html
+
+- module: tidb
+
+  slowlog:
+    enabled: true
+    # Set custom paths for the log files. If left empty,
+    # Filebeat will choose the paths depending on your OS.
+    #var.paths:
+
+  tidb:
+    enabled: true
+    # Set custom paths for the log files. If left empty,
+    # Filebeat will choose the paths depending on your OS.
+    #var.paths:
+
+  pd:
+    enabled: true
+    # Set custom paths for the log files. If left empty,
+    # Filebeat will choose the paths depending on your OS.
+    #var.paths:
+
+  tikv:
+    enabled: true
+    # Set custom paths for the log files. If left empty,
+    # Filebeat will choose the paths depending on your OS.
+    #var.paths:
+
+  tiflash:
+    enabled: true
+    # Set custom paths for the log files. If left empty,
+    # Filebeat will choose the paths depending on your OS.
+    #var.paths:
```

Comment:
Actually these logs are in the same format. Maybe not necessary to distinguish different components? (As there are also other components you missed, like CDC, TiSpark, etc)

Reply:
To distinguish between components (tidb, pd, etc...) , I had to write a lot of duplicate configurations. 
I haven't found a good solution so far.

Reply:
Is it possible to not distinguish components? I'm not sure whether log file name is enough.

Reply:
I'm not sure.
IMHO, it's better to distinguish the components to keep consistency with those components defined in TiUP cluster configs.

Reply:
I'm thinking that, by not explicitly defining the component name, the configuration process might be simplified. For example, thinking about as simple as:

```toml
log:
    var.paths: ["/data/log_dir/*/*.log"]

slowlog:
    var.paths: ["/data/log_dir/*/slow*.log"]
```

Reply:
This seems less burden of use. I'll refactor to this format.

Reply:
Some users want to add customized tags and labels to different component. In these case, different components have to separate to different fileset.
I tend to keep this unchanged for now.